src/scan.l

```diff
@@ -60,13 +60,13 @@
   char *yy_filename[ MAX_INCLUDE ];
   int yy_buf_level = 0;
   
-  char crtToken[100];
-  char nextToken[100];
+  char crtToken[1000];  /* must probably be the same as MAX_K in gdata.h */
```

Comment:
Should this just be set to MAX_K?  Or if not, another constant that can be defined up front?  I have never liked "magic numbers" for array declarations. :-)

Reply:
Also ditto for following instances of 1000

Reply:
Unfortunately, we cannot use MAX_K directly in the flex/bison files
  (*.l, *.y) because gdata.h is not included. We can, however, define
  MAX_K independently and make a note that reminds us to keep the values
  synchronized.

Reply:
Well, as I found out just now, I _can_ use MAX_K in scan.l but still _cannot_ use it in scan.y.